applications/adv_networking_bench/adv_networking_bench_default_tx_rx_loopback.yaml

```diff
@@ -0,0 +1,96 @@
+%YAML 1.2
+# SPDX-FileCopyrightText: Copyright (c) 2022-2023 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+multithreaded: true
+num_delay_ops: 32
+delay: 0.1
+delay_step: 0.01
+
+scheduler:
+  check_recession_period_ms: 0
+  worker_thread_number: 5
+  stop_on_deadlock: true
+  stop_on_deadlock_timeout: 500
+
+advanced_network:
+  cfg:
+    version: 1
+    manager: "dpdk"
+    master_core: 3
+    debug: false
+    log_level: "info"
+    loopback: true
```

Comment:
`sw_loopback`?

Reply:
That would make the feature more complicated because we'd have to differentiate between which interfaces are/aren't used in loopback, and allocate pools differently. I'm not sure we wanted that complexity yet...

Reply:
sounds good to me, top level it is. can we still nme it `sw_loopback` ?

Also, if we set this to `sw_loopback`, do we need the address on the interfaces to indicate loopback as well?

Reply:
Changed it to a string that can be "sw" or "disabled" (default)